### ISSUE TITLE
white space bar

### ISSUE THREAD

**Issue by developer0017** [dekatotoro/SlideMenuControllerSwift#362]
I have integrated your slidemenucontroller in my app,
but it has some issue. Why is there small white space?
![screen shot 2017-12-23 at 10 57 52 am](https://user-images.githubusercontent.com/17522991/34318768-3a749de4-e7d0-11e7-80d4-b6d62cf806df.png)

**Comment by soemarko** [comment]
Could be scaling issue I've just encountered.

![0820779a-aeaa-428f-841c-b023b1ab5173](https://user-images.githubusercontent.com/381585/34904523-306d1276-f87a-11e7-9477-cb3b56618624.jpg)

I've actually change the code to just resize the frame and rather than scaling it down.

<img width="219" alt="screenshot 2018-01-13 03 11 52" src="https://user-images.githubusercontent.com/381585/34904529-554cad72-f87a-11e7-8628-fe2204965e5d.png">

**Comment by ravifullestop** [comment]
There is one parameter in configuration, you can change it to 1.00 : 

    **public static var contentViewScale: CGFloat = 1.00**

**Comment by soemarko** [comment]
@ravifullestop no, I still want to scale it down. just without the white space above the navigation bar.
